tests/SeedTest.php

```diff
@@ -149,7 +149,8 @@ public function testMerge5()
         // works even if more arguments present
         $o = new ViewTestMock();
         $o->foo = ['red'];
-        $oo = $this->mergeSeeds(['foo' => ['xx']], ['foo' => ['green']], $o);
+        $oo = $this->mergeSeeds(['foo' => ['green']], $o);
```

Comment:
update needed because resolved at merge time and injected only the final one...

Reply:
But will this also work?
`$this->mergeSeeds(['foo' => ['xx']], ['foo' => ['green']], $o);` ?

Reply:
yes, but `foo` key is passed to setDefaults only once (previously this was not strict, as merged in pairs)